working-with-collections/main.js

```diff
@@ -0,0 +1,110 @@
+/* global _ */
+/* eslint-disable no-console */
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+var deck = [];
+var scores = [];
+var winningScore;
+var congrats = '';
+var players = [
+  {
+    name: 'Rachel',
+    hand: [],
+    score: 0
+  },
+  {
+    name: 'Stephen',
+    hand: [],
+    score: 0
+  },
+  {
+    name: 'Ruby',
+    hand: [],
+    score: 0
+  },
+  {
+    name: 'Kris',
+    hand: [],
+    score: 0
+  }
+];
+
+for (var i = 1; i <= 13; i++) {
+  var hearts = {};
+  hearts.rank = i;
+  hearts.suit = 'hearts';
+  hearts.value = i;
+  var spades = {};
+  spades.rank = i;
+  spades.suit = 'spades';
+  spades.value = i;
+  var diamonds = {};
+  diamonds.rank = i;
+  diamonds.suit = 'diamonds';
+  diamonds.value = i;
+  var clubs = {};
+  clubs.rank = i;
+  clubs.suit = 'clubs';
+  clubs.value = i;
+  deck.push(hearts);
+  deck.push(spades);
+  deck.push(diamonds);
+  deck.push(clubs);
+  updateDeck();
+}
+function updateDeck() {
+  for (var d = 0; d < deck.length; d++) {
+    if (deck[d].rank === 1) {
+      deck[d].rank = 'Ace';
+      deck[d].value = 11;
+    } else if (deck[d].rank === 11) {
+      deck[d].rank = 'Jack';
+      deck[d].value = 10;
+    } else if (deck[d].rank === 12) {
+      deck[d].rank = 'Queen';
+      deck[d].value = 10;
+    } else if (deck[d].rank === 13) {
+      deck[d].rank = 'King';
+      deck[d].value = 10;
+    }
+  }
+}
```

Comment:
Since the rank actually determines the point value, it's best to avoid actually storing the value on the card itself. Instead calculate the player's score by defining a function that takes a hand (array of cards) and returns the hand's point value.